src/main/java/org/simulator/examples/SBMLTestSuiteRunnerWrapper.java

```diff
@@ -0,0 +1,180 @@
+package org.simulator.examples;
+
+import org.apache.commons.math.ode.DerivativeException;
+import org.sbml.jsbml.Model;
+import org.sbml.jsbml.SBMLDocument;
+import org.sbml.jsbml.SBMLReader;
+import org.sbml.jsbml.ext.comp.CompConstants;
+import org.sbml.jsbml.validator.ModelOverdeterminedException;
+import org.simulator.comp.CompSimulator;
+import org.simulator.io.CSVImporter;
+import org.simulator.math.odes.AbstractDESSolver;
+import org.simulator.math.odes.MultiTable;
+import org.simulator.math.odes.RosenbrockSolver;
+import org.simulator.sbml.SBMLinterpreter;
+
+import javax.xml.stream.XMLStreamException;
+import java.io.*;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * <p>A wrapper class used to verify tests in the SBML Test Runner</p>
+ *
+ * <p>
+ * The results of the simulation by this class are stored as a CSV file in the specified output
+ * directory in SBML Test Runner. Then the SBML Test Runner creates the distance plots comparing
+ * this results with the pre-defined results from SBML Test Suite under the criteria given at this
+ * <a href="http://sbml.org/Software/SBML_Test_Suite/Case_Descriptions#The_.22settings.22_file" target="_blank">link</a>
+ * </p>
+ */
+public class SBMLTestSuiteRunnerWrapper {
+
+    /**
+     * Runs a simulation of a SBML file and writes result to a specified CSV file
+     * @param args
+     * @throws IOException
+     * @throws XMLStreamException
+     * @throws ModelOverdeterminedException
+     * @throws DerivativeException
+     */
+    public static void main(String[] args) throws IOException, XMLStreamException, ModelOverdeterminedException, DerivativeException {
+
+        // Configuration
+        String dirPath = args[0];
+        String currentCase = args[1];
+        String outputDirPath = args[2];
+        String level = args[3];
+        String  version = args[4];
+
+        String filePath = dirPath + '/' + currentCase + '/' + currentCase + "-sbml-l" + level + 'v' + version + ".xml";
```

Comment:
I am pretty sure this is not how file paths should be generated in Java! This will most likely break on some other os then Linux. I think one should use `File.seperator` (at least something like this is done in python and other languages). Please check how to create correct paths in Java.

Reply:
Updated using `File.separator`!